src/test/scala/cromwell/binding/ScatterSpec.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.binding
+
+import cromwell.parser.BackendType
+import cromwell.util.SampleWdl
+import org.scalatest.{FlatSpec, Matchers}
+
```

Comment:
I just thought of something that should probably be tested... and I don't think it's fully covered in these tests, and probably makes more sense in a `ScopeSpec`.  Actually, if you do create a `ScopeSpec` then some of these tests probably fit better in there.  Anyway,  the kind of test I'm thinking of is an object relationship test, like: 

```
namespace.workflow.calls.find(_.fullyQualifiedName=="w.B").parent
shouldEqual
namespace.workflow.scatters.find(_.fullyQualifiedName=="w.$scatter_0")
```